spec.md

```diff
@@ -549,14 +549,276 @@ DELETE http://localhost:8080/csp/containers/v1/snapshots/047b5b0c6a3d0ece0600000
 ```
 
 ### `/containers/v1/volume_groups`
-This endpoint is used to manage the creation and deletion of volume groups that are used for container environments.  The following methods will be supported against this endpoint.
 
- * Definitions coming soon
+This endpoint is used to manage the creation and deletion of volume groups that are used for container environments.
+
+The following methods will be supported against this endpoint.
+
+GET `/containers/v1/volume_groups`
+
+ * Return all of the volumegroups used for containers on the array.
+
+#### Request
+```
+GET http://localhost:8080/csp/containers/v1/volume_groups
+```
+
+#### Response
+
+```json
+[
+    {
+        "id" : "072265c9672660666b000000000000000000000005",
+        "name" : "volumegroup-6c58267d-b324-4b7f-88fc-6597bc62ff85",
+        "description": "my-first-volumegroup",
+        "volumes" :
```

Comment:
We need to add the **config** attribute here and to snapshotgroups like we have on the volumes and snapshots.  This will be a passthrough like it is in those cases too.